starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -1056,6 +1063,9 @@ private boolean configureVideo(
       }
 
       maybeSetMaxVideoInputSize(format);
+      if (crypto != null) {
+        flags |= MediaCodec.CONFIGURE_FLAG_USE_CRYPTO_ASYNC;
```

Comment:
`CONFIGURE_FLAG_USE_CRYPTO_ASYNC` is added in api level 34, but the change applies to all api versions.